src/core/utils/addons.js

```diff
@@ -62,3 +63,51 @@ export const getFileHash = ({
 
   return undefined;
 };
+
+export function removeUndefinedProps(object: Object): Object {
+  const newObject = {};
+  Object.keys(object).forEach((key) => {
+    if (typeof object[key] !== 'undefined') {
+      newObject[key] = object[key];
+    }
+  });
+  return newObject;
+}
+
+export const getAddonJsonLinkedData = ({
+  addon,
+}: {|
+  addon: AddonType,
+|}): Object | null => {
+  const { current_version: currentVersion, ratings } = addon;
+
+  let aggregateRating;
+  if (ratings) {
+    aggregateRating = {
+      '@type': 'AggregateRating',
+      ratingCount: ratings.count,
+      ratingValue: ratings.average,
+    };
+  }
+
+  return removeUndefinedProps({
+    '@context': 'http://schema.org',
```

Comment:
should this be: `https://schema.org/WebApplication` - in the same in the issue it has this? I see type has WebApplication tho so maybe this is okay..?

Reply:
I think that was a mistake in the issue because this does not work. I validated the schema with the Google validator.